sigma_fun/Cargo.toml

```diff
@@ -12,13 +12,16 @@ digest = "0.9"
 secp256kfun = { path = "../secp256kfun", version = "^0.3.2-alpha.0", default-features = false, optional = true }
 curve25519-dalek = { version = "3", default-features = false, optional = true, features = ["u64_backend"] }
 rand_chacha = "0.2"
+serde_crate = { package = "serde", version = "1.0", optional = true, default-features = false, features = ["derive"] }
 
 [dev-dependencies]
 rand = "0.7"
 sha2 = "0.9"
+bincode = "1"
 
 [features]
 default = ["secp256k1", "ed25519", "alloc"]
-alloc = []
+alloc = ["serde_crate/alloc"]
 secp256k1 = ["secp256kfun"]
 ed25519 = ["curve25519-dalek"]
+serde = ["serde_crate", "secp256kfun/serialization", "curve25519-dalek/serde", "generic-array/serde"]
```

Comment:
Might wanna do the same thing for the `secp256kfun` crate to avoid the `serialization` issue there. For reference, here is an explanation of this pattern: https://github.com/rust-bitcoin/rust-bitcoin/pull/373#issuecomment-689172390

Reply:
I tried for a bit, but it was giving me trouble :( I want to try again, but in a separate PR.

Reply:
Thanks #34